test/browser-plugins/plugins.test.ts

```diff
@@ -193,9 +193,6 @@ describe('Plugins', () => {
             });
 
             browser = await plugin.launch(context);
-            const argWithProxy = context.launchOptions?.args?.find((arg) => arg.includes('--proxy-server='));
```

Comment:
These expectations have been removed because we check the end IP anyway. It should not check whether it goes through a local proxy server or not. Cache checks should be separate ones.